PyCalc.py

```diff
@@ -0,0 +1,62 @@
+from math import pi, e
+from re import compile, sub
+
+
+def throwException(warning: str) -> None:
+    """Функция для отображения ошибок на консоли, вместо прерывания цикла с помощью raise"""
+    print("stderr: {}".format(warning))
+
+
+def constantCoding(expression: str) -> 'str, None':
+    """Функция проверяет выражение на наличие лишних символов и заменяет константы(pi && e)
+        (временно для проверки регулярными выражениями) на pi >> $$, e >> @ """
```

Comment:
```suggestion
    """
    Функция проверяет выражение на наличие лишних символов и заменяет константы(pi && e)
        (временно для проверки регулярными выражениями) на pi >> $$, e >> @
    """
```
3.4.2 Роз’яснення повинні починатися з трьох лапок та містити тільки ці символи, так само і кінець роз’яснення повинно закінчуватись на три лапки в окремому рядку